python/chronos/src/setup.py

```diff
@@ -17,11 +17,16 @@
 #
 
 import os
-import sys
-from shutil import copyfile, copytree, rmtree
 import fnmatch
 from setuptools import setup
 
+long_description = '''
+BigDL Chronos provides scalable time series analysis solutions using AutoML.
```

Comment:
BigDL Chronos is an application framework for building a fast, accurate and scalable time series analysis application.